libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -336,7 +336,8 @@ class RouterWrapperTests {
                 UUID,
                 1,
                 0,
-                RoutingProfile(routerOptions.profile().mapToRoutingMode(), routerOptions.user())
+                RoutingProfile(routerOptions.profile().mapToRoutingMode(), routerOptions.user()),
+                null
```

Comment:
@DenisPryt, can you please confirm that if Android SDK passes null as current geometry index, route refresh doesn't change behaviour?

Reply:
Yes, it doesn't change behaviour